.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+  - '2.7'
+
+before_install:
+  - pip install awscli
+  - "./install-apex.sh"
+
+install:
+  - "./install-requirements.sh"
+  - pip install coveralls
+
+script: coverage run test-setup.py test
+
+after_success:
+  - coveralls
+  - "./deploy.sh"
+
+env:
+  global:
+    - PYTHONDONTWRITEBYTECODE=true
```

Comment:
@PhotoNomad0 Eventually this will be deployed to Lambda from travis-ci, and the *.pyc files are not wanted.